basilisk/gui/history_msg_text_ctrl.py

```
@@ -69,7 +69,7 @@ def __init__(
 
 		self.segment_manager = MessageSegmentManager()
 		self._search_dialog = None  # Lazy initialization in _do_search
-		self.speak_stream = True
+		self.speak_response = True
```

Flake8 (7.2.0)

[warning] indentation contains tabs

(W191)